doc/js_tutorials/js_core/js_basic_ops/js_basic_ops.markdown

```diff
@@ -0,0 +1,373 @@
+@param m        output matrix; if it does not have a proper size or type before the operation, it is reallocated.
+@param rtype    desired output matrix type or, rather, the depth since the number of channels are the same as the input has; if rtype is negative, the output matrix will have the same type as the input.
+@param alpha    optional scale factor.
+@param beta     optional delta added to the scaled values.
+
+@code{.js}
+src.convertTo(dst, rtype); 
+@endcode
+
+How use MatVector
+------------------------------------
+
+@code{.js}
+let mat = new cv.Mat(); 
+// Initialise a MatVector
+let matVec = new cv.MatVector();
+// Push a Mat back into MatVector
+matVec.push_back(mat);
+// Get a Mat fom MatVector
+let cnt = matVec.get(0);
+mat.delete(); matVec.delete(); cnt.delete();
+@endcode
+
+@note Don't forget to delete cv.Mat, cv.MatVector and cnt(the Mat you get from MatVector) when you don't want to use them any more.
+
+Accessing and Modifying pixel values
+------------------------------------
+
+Firstly, you should know the following type relationship: 
+
+Data Properties  | C++ Type | JavaScript Typed Array
+---------------  | -------- | ----------------------
+data             | uchar    | Uint8Array
+data8S           | char     | Int8Array
+data16U          | ushort   | Uint16Array
+data16S          | short    | Int16Array
+data32S          | int      | Int32Array
+data32F          | float    | Float32Array
+data64F          | double   | Float64Array
+
+
+We use different ways to get the first channel of a pixel from a CV_8UC4 image. The coordinate of this pixel is (x, y).
+
+**1. data**
+
+@code{.js}
+let x = 3, y = 4;
+let src = cv.imread("canvasInput");
+if (src.isContinuous()) {
+    let pixel = src.data[y * src.cols * src.channels() + x * src.channels()];
+}
+@endcode
+
+@note  Data manipulation is only valid for continuous Mat. You should use isContinuous() to check first.
+
+**2. at**
+
+Data Properties  | At manipulation 
+---------------  | ---------------
+data             | ucharAt    
+data8S           | charAt    
+data16U          | ushortAt  
+data16S          | shortAt  
+data32S          | intAt
+data32F          | floatAt  
+data64F          | doubleAt   
+
+@code{.js}
+let x = 3, y = 4;
+let src = cv.imread("canvasInput");
+let pixel = src.ucharAt(y, x * src.channels());
+@endcode
+
+@note  At manipulation is only for single channel access and the value can't be modified.
+
+**3. ptr**
+
+Data Properties  | Ptr manipulation 
+---------------  | ---------------
+data             | ucharPtr    
+data8S           | charPtr    
+data16U          | ushortPtr 
+data16S          | shortPtr  
+data32S          | intPtr
+data32F          | floatPtr  
+data64F          | doublePtr  
+
+@code{.js}
+let x = 3, y = 4;
+let src = cv.imread("canvasInput");
+let pixel = src.ucharPtr(y, x)[0];
+@endcode
+
+mat.ucharPtr(k) get the k th column of the mat. mat.ucharPtr(i, j) get the i th column and the j row th of the mat.
```

Comment:
I understand `ptr(k)` is for kth row, and `ptr(i, j)` is for ith row and jth column?